### ISSUE TITLE
Spinner forever after unlinking a device

### ISSUE THREAD

**Issue by liliakai** [WhisperSystems/Signal-Android#4717]
Link a device, then unlink it from the device mgmt screen. A spinner is displayed and does not go away even though unlinking is complete. The user is forced tap the back arrow and select Devices again to link a new device.

Expected: Once unlinking is finished, the view re-renders the 0-1 remaining devices and the floating action button, so that the user can immediately link another device.

**Comment by agrajaghh** [comment]
duplicate of #4641  
I guess you didn't search for "wheels of confusion" :grin:

**Comment by liliakai** [comment]
I'll remember that for next time, thanks. :wink:
